requirements.txt

```diff
@@ -11,3 +11,6 @@ torch==1.7.1
 torchvision==0.8.2
 tqdm==4.65.0
 wandb==0.15.5
+transformers==4.28.0
+sentencepiece==0.1.99
```

Comment:
Running the gradio application doesn't seem to require these two packages, so please consider removing them.

Reply:
ya i just just checked, indeed not needed. I wonder why i added them